src/main/java/org/gridsuite/modification/utils/ModificationUtils.java

```diff
@@ -43,6 +43,11 @@
 // TODO remove public qualifier for all methods
 public final class ModificationUtils {
 
+    public enum Side {
```

Comment:
you don't need this 
you can use enum Side of powsyble

Reply:
True. I changed it, thank you.